tests/patcher_test.py

```diff
@@ -498,6 +498,32 @@ def test_is_daemon(self):
         self.assertEqual(lines[1], "True", lines[1])
 
 
+class Logging(ProcessBase):
+    def test_logging_early(self):
+        new_mod = """import eventlet
+eventlet.monkey_patch()
+import logging
+print(logging._lock._RLock__block.__class__)
+"""
+        self.write_to_tempfile("newmod", new_mod)
+        output, lines = self.launch_subprocess('newmod')
```

Comment:
Please, forget `write_to_tempfile` and `launch_subprocess` ever existed. Current supported testing API is `tests.run_python` and `tests.run_isolated`.